From 311d70be18d3a3ef12e93d925c64b1dc746e892d Mon Sep 17 00:00:00 2001
From: Varunkumar Allagadapa <varunkumar.allagadapa@xilinx.com>
Date: Mon, 12 Apr 2021 19:08:40 -0700
Subject: [PATCH 1791/1851] dt-bindings: soc: xilinx: Add optional property to
 vcu

commit 5f20491e8c09fb1006435b3637b8675b357025ec from
https://github.com/Xilinx/linux-xlnx.git

Add optional property "xlnx,isolation-removed" to vcu driver

Signed-off-by: Varunkumar Allagadapa <varunkumar.allagadapa@xilinx.com>
Signed-off-by: Jianqiang Chen <jianqiang.chen@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/soc/xilinx/xlnx,vcu.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Documentation/devicetree/bindings/soc/xilinx/xlnx,vcu.txt b/Documentation/devicetree/bindings/soc/xilinx/xlnx,vcu.txt
index 98474f2accca..f0a8be3e0599 100644
--- a/Documentation/devicetree/bindings/soc/xilinx/xlnx,vcu.txt
+++ b/Documentation/devicetree/bindings/soc/xilinx/xlnx,vcu.txt
@@ -44,6 +44,9 @@ Optional properties:
 - reset-gpios : The GPIO used to reset the VCU, if available. Need use this
 		reset gpio when in design 'vcu_resetn' is driven by gpio. See
 		Documentation/devicetree/bindings/gpio/gpio.txt for details.
+- xlnx,skip-isolation : This property indicates the driver that vcu gasket
+			isolation is removed by hw by default, the driver can
+			skip isolation removal.
 
 Example:
 
@@ -59,6 +62,7 @@ Example:
 		clock-names = "pll_ref", "aclk", "vcu_core_enc", "vcu_core_dec", "vcu_mcu_enc", "vcu_mcu_dec";
 		clocks = <&si570_1>, <&clkc 71>, <&xlnx_vcu 1>, <&xlnx_vcu 2>, <&xlnx_vcu 3>, <&xlnx_vcu 4>;
 		ranges;
+		xlnx,skip-isolation;
 		encoder: al5e@a0000000 {
 			compatible = "al,al5e";
 			reg = <0x0 0xa0000000 0x0 0x10000>;
-- 
2.31.1

